playbook.md

```diff
@@ -2318,3 +2318,12 @@ The process for writing a new post is:
 ### Twitter
 
 ### Facebook
+
+## Sustainability 
+We at dxw are working towards becoming a more environmentally sustainable company. We are focusing on three main areas to reduce our impact; 
+### Our Offices
+In our offices we recycle waste where possible and are in the process of switching to environmentally friendly office supplies & refreshments. We are also in the process of changing to renewable energy suppliers.
+### Our Products
+We are aware that digital services have a significant environmental impact. We are finding ways to reduce our contribution to that. This will involve hosting where possible with renewable energy and building our products efficiently. 
+### Our Behaviors
```

Comment:
Would prefer the British English spelling for consistency with the rest of the document.